public/docs/getting_started/client.md

```diff
@@ -24,9 +37,9 @@ Once launched, Truffle Develop will provide you with a console you can use to ru
 
 To read more about interacting with the console, please see the [Using the Console](/docs/getting_started/console) section.
 
-### TestRPC
+### Ganache CLI
 
-You can also use [EthereumJS TestRPC](https://github.com/ethereumjs/testrpc). It's a separate command-line blockchain you can use for both development and automated testing, and works well with Truffle. Check out its [README](https://github.com/ethereumjs/testrpc/blob/master/README.md) and documentation for more information.
+Ganache has a command-line interface, called [Ganache CLI](https://github.com/trufflesuite/ganache-cli/), with similar functionality to Ganache. This is useful for those who want to use Ganache but aren't working in a graphical environment. Ganache CLI is [available separately]((https://github.com/trufflesuite/ganache-cli/).
```

Comment:
Given the above changes, I'd reword this to pump the CLI use cases (i.e., formerly TestRPC use cases). Additionally, we don't want to say "similar functionality to Ganache" because that makes them sound like they're separate products (technically they are, but in people's heads they should remain the same, with all the same assumptions). 

I'd do:

> Ganache also has a command-line interface for those who aren't working from a graphical environment. Great for automated testing and continuous integration environments, Ganache CLI runs headless and can be configured to serve all your development needs. Ganache CLI processes transactions instantly instead of waiting for the default block time, so you can test that your code works quickly. It also tells you immediately when your smart contracts run into errors, and integrates directly with Truffle to reduce test runtime up to 90% compared to other clients. [Learn more.](https://github.com/trufflesuite/ganache-cli/)